### ISSUE TITLE
Native keyword causes yuicompressor to fail

### ISSUE THREAD

**Issue by adrian7** [niklasvh/html2canvas#522]
```
function SVGNodeContainer(node, native) {
    this.src = node;
    this.image = null;
    var self = this;
    ...
```

`native` is a reserved word in java, thus compressor will fail.

**Comment by usmonster** [comment]
I also ran into this when compiling with the Closure Compiler, but then I realized that `native` is a [_formerly_-reserved keyword](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Lexical_grammar#Future_reserved_keywords) and only affects browsers that implement ECMAScript <= 3, which basically includes only IE <= 8 and other older browsers that aren't supported by html2canvas.

The solution for Closure Compiler was to use the `--language_in=ECMASCRIPT5` command line option, so it knows not to care about earlier ECMAScript rules. There might be something similar for YUI Compressor?
